en/docs/guides/mfa/2fa-email-otp.md

```diff
@@ -56,8 +56,8 @@ This page guides you through configuring [two-factor authentication](../../../re
     CaptureAndUpdateEmailAddress = true
     showEmailAddressInUI = true
     ```
-    !!! info
-        For information on each of these configurations, see [Email OTP Configurations](../email-otp-config-advanced)
+    <!--!!! info
```

Comment:
why is this commented?

Reply:
We don't have a page with details on the advanced configurations.